projects/step-frontend/src/lib/modules/timeseries/performance-view/table/timeseries-table.component.ts

```diff
@@ -27,11 +27,14 @@ export class TimeseriesTableComponent implements OnInit, OnDestroy {
   groupDimensions: string[] = [];
 
   private keywordsService!: TimeSeriesKeywordsContext;
-  @Input() executionContext!: ExecutionContext;
+  @Input() executionContext!: TimeSeriesContext;
 
   @Output() onInitializationComplete = new EventEmitter<void>();
 
-  subscriptionsTerminator$ = new Subject<void>();
+  private terminator$ = new Subject<void>();
+
+  // @Output('onKeywordsFetched') onKeywordsFetched = new EventEmitter<string[]>();
+  // @Output('onKeywordToggled') onKeywordToggled = new EventEmitter<string>();
```

Comment:
Why did you add two commented out Outputs? Do we need them?

Reply:
No. I will clean them up. The communication is done via services now